### PR TITLE
Fix PR #16677, a parameter was missing in a function call

### PR DESCRIPTION
**Pull request by MatMaul**
I made a silly mistake in https://github.com/matrix-org/synapse/pull/16677.

Sorry for that :facepalm: 

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#changelog).
* [x] [Code style](https://element-hq.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by MatMaul** [comment]
I think the test failure is a flake.

**Review by mcalinghee — APPROVED** [review]

File: changelog.d/17033.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix PR #16677, a parameter was missing in a function call.
```
Comment (anoadragon453) [inline changelog.d/17033.bugfix:1]:
```suggestion
Fix server notice rooms not always being created as unencrypted rooms, even when `encryption_enabled_by_default_for_room_type` is in use (server notices are always unencrypted).
```

**Review by anoadragon453 — APPROVED** [review]
Apologies for taking so long to get to this; this looks good to me!

File: changelog.d/17033.bugfix (hunk 1 above)
Comment (anoadragon453) [reply changelog.d/17033.bugfix:1]: I tried to apply this suggested change myself, but I can't. So you'll need to do so before I can merge :)

**Comment by anoadragon453** [comment]
Thanks!

**Pull request merged**